app.js

```diff
@@ -53,7 +53,12 @@ addGridColors = (e) => {
 
     colorInput.forEach((color, index) => {
         const pixel = pixels[index]
-        pixel.style.backgroundColor = `#${color}`
+        if (typeof pixel === "number") {
+            pixel.style.backgroundColor = color
+        }
+        else {
+            pixel.style.backgroundColor = `#${color}`
+        }
```

Comment:
I see a couple of problems here:
1. i think you intended to check the `color` variable, not `pixel`
2. if it was a number i think you need to first convert it to the hex string, eg (`"ffcc00"`) and then prepend the `#` symbol, so it ends up like `#ffcc00`

i also have a suggestion that you don't need to use an if/else. you can instead just check if the color is a number, and if it is convert it to a string. then you can continue on with processing it in the same way as if it was a string all along.

combining all these things, this is what I think the code could look like
```suggestion
        if (typeof color === "number") {
            color = color.toString(16)
        }
        pixel.style.backgroundColor = `#${color}`
```